test/cppsuite/configs/config_poc_test_stress.txt

```diff
@@ -17,17 +17,47 @@ runtime_monitor=
         limit=100
     )
 ),
+timestamp_manager=
+(
+    enabled=true,
+    oldest_lag=10,
+    op_rate=1s,
+    stable_lag=10
+),
 workload_generator=
 (
-    collection_count=2,
-    key_count=50000,
-    key_size=10,
-    ops_per_transaction=
+    populate_config=
+    (
+        collection_count=200,
+        key_count_per_collection=5000,
+        key_size=100,
+        thread_count=20,
+        value_size=2000
+    ),
+    insert_config=
     (
-        min=5,
-        max=50
+        key_size=100,
+        op_rate=100ms,
+        ops_per_transaction=(max=100,min=0),
+        thread_count=5,
+        value_size=2000
     ),
-    read_threads=1,
-    update_threads=1,
-    value_size=2000
+    read_config=
+    (
+        op_rate=1s,
+        ops_per_transaction=(max=2000,min=100),
+        thread_count=10
+    ),
+    update_config=
+    (
+        key_size=100,
+        op_rate=15ms,
+        ops_per_transaction=(max=100,min=20),
+        thread_count=10,
+        value_size=2000
+    )
 ),
+workload_tracking=
```

Comment:
Should we add a comment why it is set false here ?

Reply:
Will do, for now its because we don't handle rollback.

Reply:
6b850f737